### ISSUE TITLE
Can't add more than 1 tab

### ISSUE THREAD

**Issue by Kekun** [sonnyp/Tangram#99]
**Tangram version**: 1.3.0 (from Flathub)

**Linux distribution**: Fedora Silverblue 33

**Flatpak version**: 1.8.2

**Desktop environment**: GNOME 3.38 on Wayland

I managed to add my first tab, then I tried to add Twitter… nope, clicking Done does nothing. Facebook then? Naha, doesn't work either.

**Comment by Kekun** [comment]
I'd love to upload a video of the process but GitHub sucks and doesn't allow that (I miss GitLab aleady).

**Comment by philn** [comment]
I wasn't able to add Twitter either. Have no FB, so haven't tried. IIRC there's a GJS stack-trace reported in the journal about Twitter.

**Comment by sonnyp** [comment]
I can reproduce, getting the following when adding twitter.com 

```sh
flatpak run re.sonny.Tangram
[...]
(re.sonny.Tangram:2): GLib-GIO-CRITICAL **: 00:19:07.829: g_input_stream_read_async: assertion 'buffer != NULL' failed
```

**Comment by sonnyp** [comment]
Problem appears because `https://twitter.com/manifest.json` doesn't reply with `content-length` HTTP header anymore.

So https://github.com/sonnyp/troll/blob/90957d2c4155b3e2b01d6e2a87e525afaeb047b0/std/fetch.js#L44 fails because `contentLength` is null.

Not sure how to best workaround this with Soup and GLib, will have a look tomorrow but if anyone has an idea...

**Comment by Kekun** [comment]
I tried adding https://gnome.element.io/ too but it shows `(re.sonny.Tangram:2): GLib-GIO-CRITICAL **: 09:02:24.848: g_input_stream_read_async: assertion 'buffer != NULL' failed`.

**Comment by philn** [comment]
> Problem appears because `https://twitter.com/manifest.json` doesn't reply with `content-length` HTTP header anymore.
> 

Checking here with `curl -v`, I do see a `content-length` response header.

**Comment by sonnyp** [comment]
@philn the response depends on the `User-Agent`.

Fixed in 1.3.1
